.changeset/three-oranges-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+[Tabs] fix inconsistent DisclosureActivator border-radius
```

Comment:
In addition to @aveline's callout, we generally don't prefix component names in square brackets in changelogs. I can't quickly find the original documentation, but I faintly remember reading docs stating to use past tense and wrap component/prop names in `` `back ticks` ``